app/scripts/modules/core/src/artifact/expectedArtifact.service.spec.ts

```diff
@@ -35,33 +35,34 @@ describe('ExpectedArtifactService', () => {
         isDefault: true,
       },
     ].map(k => ({ ...baseKindConfig, ...k }));
-    const defaultKindConfig = {
+    const customKindConfig = {
       ...baseKindConfig,
       key: 'custom',
+      isCustom: true,
```

Comment:
Is there a mismatch here or is this property name intentional? `isCustom` here vs `customKind` elsewhere.

Reply:
Good catch, I renamed it and missed this test